### PR TITLE
Adjustment setup project

### PR DESCRIPTION
**Pull request by aukraine**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 FROM ruby:2.3.6
 
-RUN apt-get update -qq && curl -sL https://deb.nodesource.com/setup_13.x | bash && apt-get install -y nodejs postgresql-client
-RUN npm install -g yarn
+RUN apt-get update -qq && curl -sL https://deb.nodesource.com/setup_12.x | bash && apt-get install -y nodejs postgresql-client && npm install -g yarnyarn install --ignore-engines
```
Comment (NickUkraine24) [inline Dockerfile]: Why you change version of node from `13.x` to `12.x`?

File: config/database.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: postgresql
   encoding: unicode
   host: db
+  port: 5432
```
Comment (NickUkraine24) [inline config/database.yml]: `Port` is by default `5432`. So, this isn't necessarily.

File: Dockerfile [hunk 3]
```diff
@@ -11,6 +11,7 @@ COPY Gemfile.lock /restaurant_rails_app/Gemfile.lock
 
 RUN bundle install
 COPY . /restaurant_rails_app
+RUN yarn install --ignore-engines
```
Comment (NickUkraine24) [inline Dockerfile]: Why you carried down after running `bundle install`?

File: Dockerfile (hunk 1 above)
Comment (aukraine) [reply Dockerfile]: you saw the old commit

File: config/database.yml (hunk 2 above)
Comment (aukraine) [reply config/database.yml]: OK, I really didn't remove this row after use db in other port

File: Dockerfile (hunk 3 above)
Comment (aukraine) [reply Dockerfile]: Because, we need to do this step after copying repo to container

File: Gemfile [hunk 4]
```diff
@@ -47,6 +47,9 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec-rails', '~> 3.9'
+  gem 'factory_bot_rails', require: false
```
Comment (NickUkraine24) [inline Gemfile]: Why `require: false`? If I not mistake, factory bot gem in the new version of RoR isn't supporting.

Comment (aukraine) [reply Gemfile]: This means install the gem, but do not call require when you start Bundler. So you will need to manually call

Comment (NickUkraine24) [reply Gemfile]: Okay, but if you know about supporting of factory-bot of the new version of RoR?

**Pull request merged**